iam.tf

```diff
@@ -672,6 +673,12 @@ data "aws_iam_policy_document" "doublecloud_airflow" {
       "arn:aws:s3:::airflow-remote-logging-*/*",
     ]
   }
+
```

Comment:
These are almost in alphabetical order, please slot it in after `iam` and before `rds`

Reply:
> Has this been tested?

Yes, together with PR to the control plane (I added you as a reviewer there as well)

> Would need to make the change to the CF template as well

Ok

Reply:
> These permissions are almost in alphabetical order, please slot it in after iam and before rds

Done